csrc/mmdeploy/core/CMakeLists.txt

```diff
@@ -54,6 +54,11 @@ target_include_directories(${PROJECT_NAME}
         $<BUILD_INTERFACE:${CMAKE_SOURCE_DIR}/third_party/json>
         )
 
+if (MSVC)
```

Comment:
Looks `Zc:preprocessor` is moved from root cmakelists here
What's the difference?